public/styles.css

```diff
@@ -0,0 +1,15 @@
+body {
+    font-family: Century Gothic, sans-serif !important;
```

Comment:
Using `!important` doesn't make me happy

Reply:
ah that was just from me testing it before we actually sorted my public folder not going to dist issue, removed it now :)